crates/holochain_conductor_api/src/app_interface.rs

```diff
@@ -100,6 +113,12 @@ pub enum AppResponse {
     /// [msgpack]: https://msgpack.org/
     ZomeCalled(Box<ExternIO>),
 
+    /// The successful response to an [`AppRequest::RevokeAgentKey`].
+    ///
+    /// The agent key has been revoked and is from now on invalid for all cells of the app. No further actions
+    /// can be authored for the cells' source chains as they are read-only now.
+    AgentKeyRevoked,
```

Comment:
At first I thought to return what the Deepkey call returns, being the action hash of the delete and the initial key registration data. But then I don't think that a happ dev needs to be concerned with that.